tswow-core/Private/TSCreature.cpp

```diff
@@ -1154,6 +1154,26 @@ void TSCreature::AttackStart(TSUnit _target)
     creature->AI()->AttackStart(target);
 }
 
+/**
```

Comment:
it's fine to keep these, but you don't need to add comments here. The ones already there are leftovers from eluna that we should probably just remove, comments in `global.d.ts` should be considered authoritative.